cosmos/comdex-test3.json

```diff
@@ -0,0 +1,91 @@
+{
+    "rpc": "https://test3-rpc.comdex.one",
+    "rest": "https://test3-rest.comdex.one",
+    "chainId": "comdex-test3",
+    "chainName": "Comdex Testnet",
+    "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/comdex-test3/chain.png",
+    "nodeProvider": {
+      "name": "Comdex Testnet Node",
+      "email": "info@comdex.one",
+      "website":"https://comdex.one"
+    },
+    "stakeCurrency": {
+      "coinDenom": "CMDX",
+      "coinMinimalDenom": "ucmdx",
+      "coinDecimals": 6,
+      "coinGeckoId": "comdex"
+    },
+    "walletUrl": "https://testnet.harborprotocol.one",
+    "walletUrlForStaking": "https://test3-explorer.comdex.one/comdex-test3/staking",
+    "bip44": {
+      "coinType": 118
+    },
+    "bech32Config": {
+      "bech32PrefixAccAddr": "comdex",
+      "bech32PrefixAccPub": "comdexpub",
+      "bech32PrefixValAddr": "comdexvaloper",
+      "bech32PrefixValPub": "comdexvaloperpub",
+      "bech32PrefixConsAddr": "comdexvalcons",
+      "bech32PrefixConsPub": "comdexvalconspub"
+    },
+    "currencies": [
+      {
+        "coinDenom": "CMDX",
+        "coinMinimalDenom": "ucmdx",
+        "coinDecimals": 6,
+        "coinGeckoId": "comdex",
```

Comment:
@dheerajkd30 Could you remove `coinGeckoId` field?
I think the price of the testnet token could be misleading to users.

Reply:
@HeesungB Updated the file as per your recommendation.